dask_cuda/explicit_comms/dataframe/shuffle.py

```diff
@@ -577,7 +577,7 @@ def wrapper(*args, **kwargs):
             kw = kw.arguments
             # Notice, we only overwrite the default and the "tasks" shuffle
             # algorithm. The "disk" and "p2p" algorithm, we don't touch.
-            if kw["shuffle"] in ("tasks", None):
+            if kw["shuffle_method"] in ("tasks", None):
```

Comment:
Does the wrapper still work if the user specifies `shuffle="tasks"`? (having trouble wrapping my head around this case)

Reply:
I _think_ we could make it work if absolutely necessary, the way things are currently written will not work if the user specifies `shuffle=`. We have not been trying to keep backwards compatibility in Dask-CUDA whatsoever.

Reply:
Ah okay. That's what I thought.

I wouldn't consider this to be a serious problem - I was just curious. The user should still see a warning anyway, so it's not like they are "silently" loosing explicit-comms support.

Reply:
The user will see a warning but only in mainline Dask, but in Dask-cuDF the `shuffle=` kwarg ceased to exist so that will error out.